packages/utils/telemetry-utils/src/test/errorLogging.spec.ts

```diff
@@ -510,8 +510,8 @@ describe("normalizeError", () => {
             errorType: "genericError",
             message,
             stack: stackHint,
-        }).withExpectedTelemetryProps({ untrustedOrigin: 1 });
-        const untrustedInputs: { [label: string]: () => { input: any; expectedOutput: TestFluidError; }; } = {
+        });
+        const testCases: { [label: string]: () => { input: any; expectedOutput: TestFluidError; }; } = {
             "Fluid Error minus errorType": () => ({
                 input: sampleFluidError().withoutProperty("errorType"),
```

Comment:
I'm wondering - should TestFluidError class extend LoggingError...?

Reply:
Eh, it's fine not to - It demonstrates that `LoggingError.typeCheck` is just looking at the shape of the object, not its prototype or instanceof or anything, for whatever that's worth.